task.py

```diff
@@ -43,11 +43,7 @@ def __init__(
         self.tc = tc
 
     def run_oc(self, cmd: str) -> host.Result:
-        if self.tenant:
-            r = self.tc.client_tenant.oc(cmd)
-        else:
-            r = self.tc.client_infra.oc(cmd)
-        return r
+        return self.tc.client(tenant=self.tenant).oc(cmd)
```

Comment:
Shouldn't this be "=="?

Reply:
no, this is correct. `tenant:bool` is a keyword argument, so this is like calling `return self.tc.client(tenant=True)`.

Except, in this case, the argument is not the `True` literal, it's `self.tenant` (which is also a bool).